onnx/version_converter/adapters/broadcast_forward_compatibility.h

```diff
@@ -28,7 +28,10 @@ class BroadcastForwardCompatibility final : public Adapter {
             n->addInput(inputs[1]);
             std::vector<int64_t> axes;
             std::vector<Dimension> new_sizes = B_sizes;
-            for (int i = 0; i < (int) (A_sizes.size() - B_sizes.size()); i++) {
+            auto size = (int) (A_sizes.size() - B_sizes.size());
```

Comment:
No C-casts. Also, please, keep sizes in size_t type.
set size to zero if B_sizes.size() is > A_sizes.size().